pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -58,15 +57,40 @@ const (
 
 var cleanUpRoutineInitialized = false
 
-var clientLock sync.Mutex
 var cleanUpRoutineInitLock sync.Mutex
 var cleanUpDummyVMLock sync.RWMutex
 
 // VSphere is an implementation of cloud provider Interface for VSphere.
 type VSphere struct {
-	conn     *vclib.VSphereConnection
-	cfg      *VSphereConfig
-	hostName string
+	cfg *VSphereConfig
+	hostName  string
+	// Stores the VSphereInstance per vcenter
+	vsphereInstanceMap map[string]*VSphereInstance
+	// Responsible for managing k8s node their location etc.
+	nodeManager *NodeManager
+}
+
+// Represents the vSphere instance
+type VSphereInstance struct {
+	conn *vclib.VSphereConnection
+	cfg  *VirtualCenterConfig
+}
+
```

Comment:
Can you add a comment which clarifies what is VirtualCenterConfig and what is VSphereConfig below.

Reply:
Done.